.buildkite/scripts/snyk/resolve_stack_version.sh

```diff
@@ -10,16 +10,10 @@ VERSION_URL="https://raw.githubusercontent.com/elastic/logstash/main/ci/logstash
 
 echo "Fetching versions from $VERSION_URL"
 VERSIONS=$(curl --silent $VERSION_URL)
-SNAPSHOTS=$(echo $VERSIONS | jq '.snapshots' | jq 'keys | .[]')
-IFS=$'\n' read -d "\034" -r -a SNAPSHOT_KEYS <<<"${SNAPSHOTS}\034"
+SNAPSHOT_VERSIONS_STR=$(echo $VERSIONS | jq -r '.snapshots | .[]')
```

Comment:
@mashhur I found it a bit simpler. If the script doesn't do the job, I am happy to close it. No rush for this sprint.